python/src/aiconfig/editor/client/package.json

```diff
@@ -33,7 +33,7 @@
     "@mantine/notifications": "^6.0.7",
     "@mantine/prism": "^6.0.7",
     "@tabler/icons-react": "^2.44.0",
-    "aiconfig": "^1.1.0",
+    "aiconfig": "../../../../../typescript",
```

Comment:
references the local folder <root>/typescript, which is the root of the aiconfig TypeScript package

Reply:
Will we need to update this when we publish the aiconfig python package next week?

Reply:
I don't think so -- `yarn build` in this folder will package up everything it needs